integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -88,7 +88,7 @@ def write_func(spark, path):
 @ignore_order
 @pytest.mark.parametrize("use_cdf", [True, False], ids=idfn)
 @pytest.mark.skipif(is_before_spark_340(), reason="Deletion vectors new in Delta Lake 2.4 / Apache Spark 3.4")
-@pytest.mark.xfail(is_databricks122_or_later(), reason="https://github.com/NVIDIA/spark-rapids/issues/8654")
+@pytest.mark.skipif(is_databricks_runtime(), reason="Separate test exists for Databricks")
```

Comment:
Odd to see a separate test for databricks.  Straightforward to create an `delta_supports_delete_vectors` boolean that can be used as a skip trigger across platforms.